app/assets/javascripts/assessments_show.coffee

```diff
@@ -8,3 +8,5 @@ $('#handin_show_assessment #fake-submit').click (e)->
 $("input[type='file']").change (e)->
 	$('#handin_show_assessment #new_submission').submit()
 	$("#handin_show_assessment input[type='file']").val("")  # clear for re-selecting the same file
+
+$('#confirmModal1').leanModal();
```

Comment:
@nayak16 did this leanModal work for you? I don't see anything pop up after clicking "release all grades". Are the script/styles required for this plugin included?